catalogs/ifpo.yaml

```diff
@@ -9,7 +9,6 @@ sources:
       set: "collection:IFPOIMAGES"
     metadata:
       data_path: ifpo/photographs
-      post_harvest: "add_thumbnail_urls"
```

Comment:
Removing the post_harvest step from this collection but keeping an example of post_harvest in the DASI collection added below.